easystepper/easystepper.go

```diff
@@ -26,31 +31,97 @@ func New(pin1, pin2, pin3, pin4 machine.Pin, steps int32, rpm int32) Device {
 	}
 }
 
+// NewDual returns a new dual easystepper driver given 8 pins numbers (not pin object),
+// number of steps and rpm
+func NewDual(pin1, pin2, pin3, pin4, pin5, pin6, pin7, pin8 machine.Pin, steps int32, rpm int32) DualDevice {
+	pin1.Configure(machine.PinConfig{Mode: machine.PinOutput})
```

Comment:
i would remove these from the `New` and `NewDual()` functions, and instead add `Configure()` method like this:

```go
// Configure sets up the device.
func (d *Device) Configure() {
	for _, pin_ := range d.pins {
		pin.Configure(machine.PinConfig{Mode: machine.PinOutput})
	}
}
```

Then you can add one to `DualDevice` like this:

```go
// Configure sets up the device.
func (dd *DualDevice) Configure() {
	dd.devices[0].Configure()
	dd.devices[1].Configure()
}
```